src/match/match.controller.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  Body,
+  Controller,
+  Delete,
+  Get,
+  Param,
+  Patch,
+  Post,
+} from '@nestjs/common';
+import { MatchIdValidationPipe } from './validationPipes/MatchIdValidationPipe';
+import { CreateMatchValidationPipe } from './validationPipes/CreateMatchValidationPipe';
+import { MatchService } from './match.service';
+import { PatchMatchDto, CreateMatchDto } from './match.dto';
+import { PatchMatchValidationPipe } from './validationPipes/PatchMatchValidationPipe';
+
+@Controller('match')
```

Comment:
@ApiTags('Matches') for swagger purposes. You can do this in all of your controllers